.github/scripts/prepare_data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
:wrench:  make the script fail if things go wrong

```suggestion
#!/usr/bin/env -S bash -eux -o pipefail
```